core/templates/dev/head/css/oppia.css

```diff
@@ -680,10 +680,17 @@ textarea {
   height: 198px;
 }
 
+.oppia-footer-margin{
```

Comment:
I thought for media queries you can add css styles for classes that exists like for eg ```oppia-footer-padding ``` 
If I add a class that didn't exist on div for footer will it get add if the smaller size is hit?
help me because I don't know :sweat_smile: